widget/src/scrollable.rs

```diff
@@ -650,6 +646,10 @@ where
             }
         }
 
+        if let event::Status::Captured = event_status {
+            return event::Status::Captured;
+        }
+
         event_status
```

Comment:
Didn't look much into the logic, but, assuming it is correct, is the early return still necessary? Perhaps we can just return `event_status` normally now, even if it's `Captured`.

Reply:
I think that while the functionality of the scrollable widget remains unaffected by the returned state value, it's still crucial to discern which event is being processed by the widget. In this scenario, the widget is actively listening to events, adjusting values that directly influence its behavior, that's why the return state being marked as Captured.

```
/// The status of an [`Event`] after being processed.
#[derive(Debug, Clone, Copy, PartialEq, Eq)]
pub enum Status {
    /// The [`Event`] was **NOT** handled by any widget.
    Ignored,

    /// The [`Event`] was handled and processed by a widget.
    Captured,
}
```

Reply:
Yeah, sorry, I should have been clearer. I mean that, here, you're returning `event::Status::Captured` if `event_status` is `event::Status::Captured`; otherwise, you return `event_status`. But that's the same as just returning `event_status`.

In other words, the following would likely have the same behavior (unless there are side effects I'm not seeing here):

```suggestion
        event_status
```

Reply:
Oh yeah, you are right. I was thinking the same when I was checking the logic again some few seconds ago. Thanks for noticing. I'm going to make the commit.